### ISSUE TITLE
Method getScrollY don't have types

### ISSUE THREAD

**Issue by eylonxx** [alvarotrigo/react-fullpage#411]
Basically title.
Trying to access fullPageAPI.getScrollY() is failing.
It's not in the index.d.ts either but when console.logging fullPageApi object (which is provided in the render function), you can actually see it defined there.

**Comment by alvarotrigo** [comment]
The type is not defined, but the function is there.

If you replace the render function for this one in [this example](https://github.com/alvarotrigo/react-fullpage/blob/master/examples/react/src/hooks.js) you'll see how it works as expected:

```jsx
render={({ state, fullpageApi }) => {
          return (
          <ReactFullpage.Wrapper>
            {fullpages.map(({ text }) => (
              <div key={text} className={SEL}>
                <div className="slide">
                  <h1 onClick={() => console.log(fullpageApi.getScrollY())}>a</h1>
                  <a href="https://google.com">fadsfad</a>
                </div>
                <div className="slide">
                  <h1>b</h1>
                </div>
                <div className="slide">
                  <h1>c</h1>
                </div>
              </div>
            ))}
          </ReactFullpage.Wrapper>
        )}}
```

**Comment by alvarotrigo** [comment]
Fixed in version 0.1.39! 🥳
